editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -258,6 +263,7 @@ export function updateInteractionViaDragDelta(
       userPreferredStrategy: currentState.userPreferredStrategy,
       startedAt: currentState.startedAt,
       updatedTargetPaths: currentState.updatedTargetPaths,
+      aspectRatioLock: null,
```

Comment:
I think this shouldn't reset the aspectRatioLock, and just use currentState.aspectRatio lock. same with the other `update...` functions

Reply:
oops, fixed :)